styles.css

```diff
@@ -1 +1,212 @@
-body
\ No newline at end of file
+@import url('https://fonts.googleapis.com/css2?family=Merriweather&family=Roboto&display=swap');
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+body {
+    font-family: "Roboto", Arial, sans-serif;
```

Comment:
Goed dat je nog een fallback font gebruikt hebt en TOP dat je het font in de body declareert!